crates/bevy_render/src/mesh/shape/regular_polygon.rs

```diff
@@ -4,7 +4,7 @@ use wgpu::PrimitiveTopology;
 /// A regular polygon in the `XY` plane
 #[derive(Debug, Copy, Clone)]
 pub struct RegularPolygon {
-    /// Inscribed radius in the `XY` plane.
+    /// Circumscribed radius in the `XY` plane.
```

Comment:
```suggestion
    /// Circumscribed radius in the `XY` plane.
    /// 
    /// In other words, this polygon will completely fit inside a circle of this radius.
```

Reply:
How about this for increased precision? 
```suggestion
    /// Circumscribed radius in the `XY` plane.
    /// 
    /// In other words, the vertices of this polygon will all touch a circle of this radius.
```